### PR TITLE
Adding ExposeSampleData field to web service properties

### PR DESCRIPTION
**Pull request by tudordid**
- regenenerated sdk classes after addition of ExposeSampleData in web service properties
- changed UTs to create service's storage account dynamically
- re-recorded UTs

This is related to this change in the swagger spec: https://github.com/Azure/azure-rest-api-specs/pull/301

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@tudordid__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (tudordid). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/ResourceManagement/MachineLearning/MachineLearning.Tests/ScenarioTests/WebServiceTests.cs [hunk 1]
```diff
@@ -55,12 +59,12 @@ public class WebServiceTests : TestBase
 
         private const int AsyncOperationPollingIntervalSeconds = 5;
 
-        private delegate void AMLWebServiceTestDelegate(string webServiceName, string resourceGroupName, ResourceManagementClient resourcesClient, AzureMLWebServicesManagementClient amlServicesClient, string cpResourceId);
-        
+        private delegate void AMLWebServiceTestDelegate(string webServiceName, string resourceGroupName, ResourceManagementClient resourcesClient, AzureMLWebServicesManagementClient amlServicesClient, string cpResourceId, StorageAccount storageAccount);
```
Comment (markcowl) [inline src/ResourceManagement/MachineLearning/MachineLearning.Tests/ScenarioTests/WebServiceTests.cs]: Please fix line breaks - should break ~ 80 columns

**Comment by markcowl** [comment]
@tudordid One nit, otherwise, looks good.  Please fix and we will merge.

File: src/ResourceManagement/MachineLearning/MachineLearning.Tests/ScenarioTests/WebServiceTests.cs (hunk 1 above)
Comment (tudordid) [inline src/ResourceManagement/MachineLearning/MachineLearning.Tests/ScenarioTests/WebServiceTests.cs]: fixed

**Pull request merged**